test/e2e/spec/oidc_provider/client_spec.js

```diff
@@ -0,0 +1,67 @@
+describe("OpenId Provider", function() {
+    
+    var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js');
+    var expect = chakram.expect;
+    var DOMAIN = 'https://' + process.env.BOILERPLATE_DOMAIN;
+    var URL = DOMAIN + '/api/openid';
+
+    describe("provides an openid-configuration", function() {
+
+        it("as json", function() {
+            var response = chakram.get(URL + '/.well-known/openid-configuration');
+            return expect(response).to.have.header('Content-Type', 'application/json');
+        }); 
+
+        it("with provider information", function() {
+            var response = chakram.get(URL + '/.well-known/openid-configuration');
+            expect(response).to.have.json('issuer', URL);
+            expect(response).to.have.json('authorization_endpoint', URL + '/authorize');
+            expect(response).to.have.json('jwks_uri', URL + '/jwks');
+            return chakram.wait();
+        }); 
+    });
+
+    describe("reacts on an authorization request", function () {
+
+        it("with redirect to login page if client_id, redirect_uri, state, once, response_type and scope exists and valid", function () {
```

Comment:
I don't understand this test. Redirect functionality is added by #48.

Reply:
If so, then this test should fail, shouldn't it?

Reply:
IMHO this checks the redirect to the default destination as provided with the plugin, so it is fine.
@nielsgroth correct me if I am wrong